samples/tacos/src/main/kotlin/com/slack/circuit/tacos/MainActivity.kt

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2022 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+package com.slack.circuit.tacos
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material3.MaterialTheme
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+import com.slack.circuit.foundation.CircuitCompositionLocals
+import com.slack.circuit.foundation.CircuitConfig
+import com.slack.circuit.foundation.CircuitContent
+import com.slack.circuit.runtime.presenter.Presenter
+import com.slack.circuit.runtime.ui.Ui
+import com.slack.circuit.runtime.ui.ui
+import com.slack.circuit.tacos.repository.IngredientsRepositoryImpl
+import com.slack.circuit.tacos.step.FillingsProducerImpl
+import com.slack.circuit.tacos.step.ToppingsProducerImpl
+import com.slack.circuit.tacos.step.confirmationProducer
+import com.slack.circuit.tacos.step.summaryProducer
+import com.slack.circuit.tacos.ui.theme.TacoTheme
+
+class MainActivity : AppCompatActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    val circuitConfig: CircuitConfig =
+      CircuitConfig.Builder()
+        .addPresenterFactory(buildPresenterFactory())
+        .addUiFactory(buildUiFactory())
+        .build()
+
+    setContent {
+      TacoTheme {
+        rememberSystemUiController().run { setStatusBarColor(MaterialTheme.colorScheme.background) }
```

Comment:
```suggestion
        rememberSystemUiController().setStatusBarColor(MaterialTheme.colorScheme.background)
```

Reply:
I don't think so, but may be unaware of something. This is using Accompanist to set the system status bar colour, not the app top bar colour. I don't think the scaffold can set that

Reply:
the top app bar in M3 can fill into the status bar and then all we should normally have to do is set the status bar icons to be light or dark